Tests/SwiftMemcacheTests/UnitTest/MemcachedResponseDecoderTests.swift

```diff
@@ -94,4 +102,36 @@ final class MemcachedResponseDecoderTests: XCTestCase {
         var buffer = self.makeMemcachedResponseByteBuffer(from: notFoundResponse)
         try self.testDecodeResponse(buffer: &buffer, expectedReturnCode: .NF)
     }
+
+    func testDecodeValueResponse() throws {
```

Comment:
Yeah we need a couple of more tests here that are trying to pass partial responses and assert that we return the right things.